crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -1397,11 +1399,58 @@ mod tests {
     use crate::error::{DevnetResult, Error};
     use crate::starknet::starknet_config::{StarknetConfig, StateArchiveCapacity};
     use crate::state::state_diff::StateDiff;
-    use crate::traits::{Accounted, HashIdentified};
+    use crate::traits::{Accounted, Deployed, HashIdentified};
     use crate::utils::test_utils::{
-        dummy_contract_address, dummy_declare_transaction_v1, dummy_felt,
+        cairo_0_account_without_validations, dummy_contract_address, dummy_declare_transaction_v1,
+        dummy_felt,
     };
 
+    /// Initializes starknet with 1 account that doesn't require tx without validation and with the
+    /// specified state archive capacity
+    pub(crate) fn setup_starknet_with_unvalidated_account_and_state_capacity(
```

Comment:
unvalidated_account doesn't reflect the real property of the account. I can propose a longer name setup_starknet_with_no_signature_check_account_and_state_capacity

Reply:
Fixed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/494/commits/896b8cf81536b0dce9fae2d5a9fc95c7b037a9a3